src/resources/js/components/ListPresentations.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="container">
+    <div v-if="showMessage">
+      <h3>{{ message }}</h3>
+    </div>
+
+    <div v-for="presentation in presentations.data" :key="presentation.id">
+      <router-link
+        :to="{ name : 'presentation', params : { idPresentation : presentation.id} }"
+        >
+        {{ presentation.title }}
+      </router-link>
+    </div>
+
+    <router-view></router-view>
```

Comment:
Why router-view is included here ?

Reply:
it is an error I've remove it in [e98bec](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/115/commits/e98bec201a16cfa04155cf69b3595941d2784b36)